cmd/user-api/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/afernandowex/ps-tag-onboarding-go/internal/app/mysql"
+	"github.com/afernandowex/ps-tag-onboarding-go/internal/app/repository"
+	"github.com/afernandowex/ps-tag-onboarding-go/internal/app/user-api/controller"
+	"github.com/afernandowex/ps-tag-onboarding-go/internal/app/user-api/routing"
+	"github.com/afernandowex/ps-tag-onboarding-go/internal/app/user-api/service"
+	"github.com/afernandowex/ps-tag-onboarding-go/internal/app/user-api/validation"
+	"github.com/labstack/echo/v4"
+)
+
+func main() {
+	db := mysql.InitialiseMySQL()
```

Comment:
I do think this main is too cluttered for what a main should be, but I understand it is a simple service right now. If this was expanded, it would be worth it to separate these out.

Reply:
Seems like this is a point of contention. I've extracted this to a separate method at this time.